src/components/TodoList/TodoList.tsx

```diff
@@ -1,12 +1,27 @@
 /* eslint-disable */
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { getSortedTodos } from '../../helpers/getSortedTodos';
+
+type TodoListProps = {
+  loading: boolean;
+};
+
+export const TodoList: React.FC<TodoListProps> = ({ loading }) => {
+  const dispatch = useAppDispatch();
+  const todos = useAppSelector(state => state.todos);
+  const { status, query } = useAppSelector(state => state.filter);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const filteredTodos = getSortedTodos(todos, { status, query });
 
-export const TodoList: React.FC = () => {
   return (
     <>
-      <p className="notification is-warning">
-        There are no todos matching current filter criteria
-      </p>
+      {!loading && !todos.length && (
```

Comment:
```suggestion
      {(!loading && !todos.length) && (
```
If you don't use constant for several conditions try to use brackets for it. 
In the future, a person can simply add an additional condition based on a previous condition. ( not critical)